console/frontend/src/main/frontend/src/app/components/pages/sidebar.service.ts

```diff
@@ -4,32 +4,37 @@ import { Injectable } from '@angular/core';
   providedIn: 'root',
 })
 export class SidebarService {
+  private bodyElement = document.querySelector<HTMLElement>('body')!;
+  private sideMenuElement = document.querySelector<HTMLElement>('#side-menu');
```

Comment:
If this service would be changed to a directive this could be gathered with @ViewChild. Whould that be possible?

Reply:
Yes that should work